src/main/java/com/squareup/javapoet/MethodSpec.java

```diff
@@ -376,8 +413,9 @@ public Builder defaultValue(CodeBlock codeBlock) {
     }
 
     /**
-     * @param controlFlow the control flow construct and its code, such as "if (foo == 5)".
-     * Shouldn't contain braces or newline characters.
+     * @param controlFlow
+     *          the control flow construct and its code, such as "if (foo == 5)". Shouldn't contain
```

Comment:
nit: our code style is that this all goes on the same line, with wrapped lines +4